### DISCUSSION TITLE
How to add pdf?

### DISCUSSION THREAD

**Discussion by elaboratebriefly** [TriliumNext/Trilium#2334] [Q&A] [answered]
Hello!
How can I attach pdf file to a note?
`/home/user/Documents/file.pdf` does not work.

Trilium v0.48.5 on linux.
Thanks!

**Comment by zadam** [comment, marked as answer]
Drag & drop the PDF file into a note, or e.g. 
![image](https://user-images.githubusercontent.com/617641/141658685-7889bea1-2d3e-49fb-ac89-bbe496a87e4e.png)

**Reply by elaboratebriefly** [reply]
Thank you!
